src/Microsoft.OpenApi.OData.Reader/Edm/ODataKeySegment.cs

```diff
@@ -77,13 +77,7 @@ public override string GetPathItemName(OpenApiConvertSettings settings, HashSet<
                 }
                 else
                 {
-                    IList<string> keyStrings = new List<string>();
```

Comment:
Again, I have a different opinion for the single quote.

Single quote is part of the string, enum value, it goes with value. But not for the template.

When Graph SDK generate the URL (the real URL), should generate the single quote for string type. 
For example:

'8' for string type, but 8 for int type. 

So, the changes in this PR looks "invalid" to me. I'd like @mikepizzo's input.

Reply:
I disagree. This quotes behavior comes from odata conventions. The open api spec does not mandate any quote for url parameters, regardless of the types.
The point of this library is to encapsulate the odata conventions away.
This is why I believe the quotes should be added during the conversion process.